QueryManager/src/main/java/eu/neclab/ngsildbroker/queryhandler/repository/QueryDAO.java

```diff
@@ -61,18 +58,18 @@ public class QueryDAO {
 
 	public Uni<Map<String, Object>> getEntity(String entityId, String tenantId, AttrsQueryTerm attrsQuery) {
 		return clientManager.getClient(tenantId, false).onItem().transformToUni(client -> {
-			String sql = "";
-			sql += "SELECT ENTITY";
-
-			Tuple tuple = Tuple.tuple();
+			String sql = "SELECT ";
 			if (attrsQuery != null) {
-				sql += "-$1 FROM ENTITY WHERE ID=$2";
-				tuple.addArrayOfString(attrsQuery.getAttrs().toArray(new String[0]));
+				for (String attrs : attrsQuery.getAttrs()) {
```

Comment:
this is a fix for attrs query. please add a test to the test suite.
also is this properly working? what about id, type, scope, createdAt and modifiedAt should be added as well

Reply:
yes, added